### PR TITLE
BXC-3190 - Reorganize persistence module

### PR DESCRIPTION
**Pull request by bbpennel**
https://jira.lib.unc.edu/browse/BXC-3190

* Added operations, operations-jms, deposit-utils modules
    * Moved appropriate classes into these modules
    * Purpose of persistence module refocused on persistence of binaries. Added persistence-api module to prevent dependency issues
* Cleaned up dependencies, unused code, unused resources. Removed some usages of classes from activemq that should have been from base java.

### CODE REVIEW

File: services/src/test/resources/spring-test/cdr-client-container.xml [hunk 1]
```diff
@@ -186,8 +184,4 @@
         class="org.springframework.web.multipart.commons.CommonsMultipartResolver">
         <property name="maxUploadSize" value="2000000000"/>
```
Comment (lfarrell) [inline services/src/test/resources/spring-test/cdr-client-container.xml:185]: Do we want this to be hard coded?

**Review by lfarrell — CHANGES_REQUESTED** [review]
One question, unrelated to the pr.

File: services/src/test/resources/spring-test/cdr-client-container.xml (hunk 1 above)
Comment (bbpennel) [reply services/src/test/resources/spring-test/cdr-client-container.xml:185]: Good point, I've created a ticket https://jira.lib.unc.edu/browse/BXC-3222

Comment (lfarrell) [reply services/src/test/resources/spring-test/cdr-client-container.xml:185]: See https://jira.lib.unc.edu/browse/BXC-3222

**Review by lfarrell — APPROVED** [review]

**Pull request merged**